### DISCUSSION TITLE
I need help with my application for ios

### DISCUSSION THREAD

**Discussion by jose19215** [appcelerator/titanium_mobile#13847] [Q&A]
I get this error

ERROR] warning: conditional compilation flags do not have values in Swift; they are either present or absent (rather than 'TI_VERSION=12.0.0')
[ERROR] ** BUILD FAILED **
[ERROR] The following build commands failed:
[ERROR] Ld/Users/user1/Document/Programador/build/iphone/build/Products/Debug-iphonesimulator/Programador.app/Programador normal (in target 'Programador' from project 'Programador')
[ERROR] (1 failure)
 
Process exited with 1
 
Titanium version 12.1.2 GA

**Comment by m1ga** [comment]
just to verify since the error message still says `12.0.0`: you've updated the version in tiapp.xml too?

**Reply by m1ga** [reply]
just checking in again, is it still an open issue @jose19215 ?

**Comment by jose19215** [comment]
Good afternoon, I already updated and it still appears

**Reply by m1ga** [reply]
Do you use any external module? We have a user with the same error using Xcode 14.3.x because a native library in the module was compiled with an older XCode version and the Swift versions don't match up. 

So if you can try to downgrade to Xcode 14.2.x
